packages/api-core/src/index.ts

```diff
@@ -35,6 +35,7 @@ export * from './site';
 export * from './site-activity-log';
 export * from './site-address-change';
 export * from './site-atomic-transfers';
+export * from './site-automated-transfers-eligibility';
```

Comment:
Curious that we have both `atomic-transfers` and `automated-transfers`. I kind of wonder if they belong in the same folder 🤷‍♂️ I guess we've been matching 1:1 with API routes though.

Reply:
No strong opinions, happy to move it if we prefer to do so 🙂